### ISSUE TITLE
Расчетный баланс в отряде

### ISSUE THREAD

**Issue by hippogamesunity** [hippogamesunity/SpellMasters#25]
Сейчас у нас имеются следующие стартовые персонажи: 2 воина, 2 лучника, 2 боевых мага и 2 бафера. Dummy и Slave являются редкими уникальными юнитами, поэтому я их пока не в расчет не беру. Как мне смутно представляется, баланс в отряде должен быть следующим: 40% воины, 30% лучники и 30% маги, т.е. воинов должно быть больше. Исходя из этой схемы будут определяться интервалы при генерации юнитов.
Еще я предлагаю добавить отдельную группу воинов с двуручным оружием, чтобы обеспечить расчетные 40% воинов ближнего боя без чрезмерного увеличения интервалов ополченца и наемника.

**Comment by aepavlov** [comment]
Я в принципе  со схемой согласен, на мой взгляд больше всего ролей должно быть у воинов и/или магов. Со стрелками как-то все проще.

**Comment by Ragewortt** [comment]
В общем-то я тоже согласен. На счет юнита с двуручным оружием. Второе оружие будет вместо щита чисто визуально, а урон будет двойной?

**Comment by hippogamesunity** [comment]
Нет, двуручники у нас сейчас в развитии ополченца и наемника - это ветки копейщика и берсерка. Из них можно сделать отдельную группу, но она сможет пересекаться с ополченцем и наемником.
А двойное оружие у нас у гладиатора, правда ни анимации пока нет, ни возможности взять второй меч в редакторе.

**Comment by hippogamesunity** [comment]
Обновил раздел с воинами ближнего боя
https://github.com/hippogamesunity/SpellMasters/wiki/%5B%D0%AE%D0%BD%D0%B8%D1%82%D1%8B%5D-%D0%92%D0%BE%D0%B8%D0%BD%D1%8B-%D0%B1%D0%BB%D0%B8%D0%B6%D0%BD%D0%B5%D0%B3%D0%BE-%D0%B1%D0%BE%D1%8F

**Comment by aepavlov** [comment]
Не совсем понял зачем рабы, тем более слабые. По идее редкими должны быть наоборот крутые перцы. А вот копейщики-алебардщики - классная идея.

**Comment by hippogamesunity** [comment]
Рабы потом превращаются в сильных гладиаторов.
